package.json

```diff
@@ -11,7 +11,7 @@
     "@mapbox/geojson-extent": "^0.3.2",
     "@onaio/connected-private-route": "^0.0.8",
     "@onaio/connected-reducer-registry": "^0.0.1",
-    "@onaio/drill-down-table": "^0.0.20",
+    "@onaio/drill-down-table": "^0.0.21",
```

Comment:
Had to upgrade this component to support our usage, as seen in the `IRSTableCell` component